Casks/d/djstudio.rb

```diff
@@ -0,0 +1,39 @@
+cask "djstudio" do
+  arch arm: "-arm64"
+
+  on_mojave :or_older do
+    version "2.6.28"
+    sha256 "a24739ad73b5fbda7c64aaae320b1486622c669ba3dc414d692def6026d2639d"
+
+    url "https://github.com/AppMachine/dj-studio-app-updates/releases/download/v#{version}/DJ.Studio-#{version}.dmg"
+  end
+  on_catalina :or_newer do
+    version "2.6.104"
+    sha256 arm:   "8f6fb08230fb15044f83577ec4ec32735a4983cfed5e3a0cf9c4342dd38a55b7",
+           intel: "cbb417f2802c0221b91d857c610fd583bfa9face941c8337c90977e1d0f98f5e"
+
+    url "https://download.dj.studio/DJ.Studio-#{version}#{arch}.dmg"
+  end
+
```

Comment:
```suggestion
  on_mojave :or_older do
    version "2.6.28"
    sha256 arm:   "5733f6b7014f5c21ffce647fd9b63aaf96e7852e3953d3dbf825641a1ce006c1",
           intel: "a24739ad73b5fbda7c64aaae320b1486622c669ba3dc414d692def6026d2639d"
  end
  on_catalina :or_newer do
    version "2.6.104"
    sha256 arm:   "8f6fb08230fb15044f83577ec4ec32735a4983cfed5e3a0cf9c4342dd38a55b7",
           intel: "cbb417f2802c0221b91d857c610fd583bfa9face941c8337c90977e1d0f98f5e"
  end

  url "https://github.com/AppMachine/dj-studio-app-updates/releases/download/v#{version}/DJ.Studio-#{version}#{arch}.dmg",
      verified: "github.com/AppMachine/dj-studio-app-updates/"
```

Since the current releases are also published in the upstream repository, we can use the same `url` for both versions.

However, the `on_os` blocks might be incorrect, as an ARM binary exists for both versions.

Reply:
The arm binary is relevant for the `on_mojave` block, because Big Sur is the oldest software that is supported on arm.

Reply:
True, that's why I thought the `on_os` blocks were incorrect. I will update the suggestions accordingly.

Reply:
Since arm binary is not relevant for older macOS versions as @bevanjkay wrote, I am not sure how could I change this code to use the same URL, without adding irrelevant sha256 for arm.

Would this work?

```
  on_mojave :or_older do
    version "2.6.28"
    sha256 intel:  "a24739ad73b5fbda7c64aaae320b1486622c669ba3dc414d692def6026d2639d"
  end
```

If this would work, I will update the cask later.